mse_home/template/secrets.json

```diff
@@ -1,3 +1,3 @@
 {
-    "key": "00112233445566778899aabbccddeeff"
+    "sk": "PVyaHlCyN40t5YeAgqLxbz8aISYg1QbRN4qyo_QqIlM="
```

Comment:
"password"?

Reply:
Fixed in ffdad282e7d508216010f865077d045ddc712c2a.